src/components/style/ModalContent.css

```diff
@@ -0,0 +1,81 @@
+.modal-display {
+  background-color: white;
+  position: absolute;
+  align: center;
+  top: 32%;
+  border-radius: 6px;
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
I was expecting the modal to work on mobile.

> The modal should be the same width as the tweet list both on desktop and mobile sizes

Perhaps this line was confusing. Anyway, if you look at the tweet list in chrome dev tools with device emulation. You'll see the tweets container `.search-results` switches to a 100% width style in a media query. Can you add a styles for the same media breakpoint `@media (max-width: 500px)` that will adjust the width of the modal. You may need to also target some of the modal contents to change the layout to something that works better on mobile.